src/main/perf/Indexer.java

```diff
@@ -617,6 +619,11 @@ public DocValuesFormat getDocValuesFormatForField(String field) {
       PerfUtils.clearDir(dir);
       PerfUtils.copyDir(tmpDir, dir);
       tmpDir.close();
+      Files.walk(tmpDirPath)
```

Comment:
Should we maybe move this into a `try/finally` clause?  And maybe print that we are deleting this path now?

Reply:
And could you fix the above `System.out.println(tmpDirPath)` to say a bit more e.g. `Using temp directory "..." to rearrange`?

Reply:
Good ideas! Done